tests/service/ExprTests.fs

```diff
@@ -22,6 +23,60 @@ let internal exprChecker = FSharpChecker.Create(keepAssemblyContents=true)
 
 [<AutoOpen>]
 module internal Utils = 
+    let getTempPath() = 
+        Path.Combine(Path.GetTempPath(), "ExprTests")
+
+    /// If it doesn't exists, create a folder 'ExprTests' in local user's %TEMP% folder
+    let createTempDir() = 
+        let tempPath = getTempPath()
+        do 
+            if Directory.Exists tempPath then ()
+            else Directory.CreateDirectory tempPath |> ignore
+
+    /// Returns the filename part of a temp file name created with Path.GetTempFileName().
+    let getTempFileName() =
```

Comment:
If you delete the file created with CreateTemp file, then the next time getTempFileName is called you will get the exact same filename, which means if the tests are run multi threaded, there is a fair chance that you will get filename collisions.

So .. a simpler approach would be to just compute a guid.ToString(), or combine  the file name with process id and thread id.

Reply:
@KevinRansom Interesting, I wasn't sure of that, as when testing, it appeared to be a different name each time (assuming it calls this, which is based on system time: https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettempfilenamea). But that still leaves the multi threading issue. 

But that guarantees is not given here: https://docs.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename?view=netcore-3.1. Which, btw, also suggests that cleaning up is a good thing, as after 64k files it'll raise an error.

So that function is not as useful as one might think. I'll fix it.

Reply:
I've fixed it (better safe than sorry, and best practices and whatnot), but I don't think the risk was so big, as when you delete it, the same name is _not_ chosen again:

```f#
> Path.GetTempFileName();;
val it : string = "C:\Users\Me\AppData\Local\Temp\tmpB0EB.tmp"

> File.Delete it;;
val it : unit = ()

> Path.GetTempFileName();;
val it : string = "C:\Users\Me\AppData\Local\Temp\tmpDFD8.tmp"

> File.Delete it;;
val it : unit = ()

> Path.GetTempFileName();;
val it : string = "C:\Users\Me\AppData\Local\Temp\tmpB051.tmp"

> File.Delete it;;
val it : unit = ()
```